internal/service/timestreaminfluxdb/db_instance.go

```diff
@@ -215,6 +211,18 @@ func (r *resourceDBInstance) Schema(ctx context.Context, req resource.SchemaRequ
 					also use the InfluxDB CLI to create an operator token. These attributes will be 
 					stored in a Secret created in AWS SecretManager in your account.`,
 			},
+			names.AttrPort: schema.Int64Attribute{
+				Optional: true,
+				Computed: true,
```

Comment:
would it be redundant to add default here as well?
```suggestion
				Computed: true,
				Default: int64(8086),
```

Reply:
It would be redundant, the port defaults to `8086` anyways when creating an instance.